servers/shares-api/schema.graphql

```diff
@@ -1,3 +1,77 @@
-type PocketShare {
-  id: ID!
-}
\ No newline at end of file
+extend schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")
+  @link(
+    url: "https://specs.apollo.dev/federation/v2.3"
+    import: ["@key", "@composeDirective", "@tag"]
+  )
+  # The link directive is required to federate @constraint
+  # It doesn't actually have to be a real spec, but it would be good
+  # to write one and replace this.
```

Comment:
Surprised the library we use doesnt have one up

Reply:
eh the readme of the repo probably counts 😅

Reply:
I tried this but the link directive requires a specific URL which does not pass with a semver like `vX.X.X` (requires `vX.X`)